cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/optimized/messages/MessagesBodyReq.kt

```diff
@@ -8,7 +8,7 @@ import kotlinx.serialization.json.JsonObject
 @Serializable
 internal data class MessagesBodyReq(
     @SerialName("accountId")
-    val accountId: Long,
+    val accountId: Int,
```

Comment:
Are you sure every single `accountId` can fit into 32-bit signed integer instead of 64-bit?

Reply:
Yes, account ids are small numbers.